templates/deployment.yaml

```diff
@@ -9,6 +17,12 @@ spec:
   selector:
     matchLabels:
       {{- include "krakend.selectorLabels" . | nindent 6 }}
+  {{- if eq .Values.deploymentType "rollout" }}
```

Comment:
This is a [valid section](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy) for a k8s apps/v1/Deployment. Id remove this conditional

Reply:
oho! that's good to know!